packages/web/src/scss/components/Card/_Card.scss

```diff
@@ -76,77 +51,97 @@
 
-    &:has(.CardMedia:not(:only-child)) {
-        padding-right: var(--#{tokens.$css-variable-prefix}card-padding);
-    }
-}
+    @include breakpoint.up($breakpoint-value) {
+        .Card--#{$infix}vertical {
+            --#{tokens.$css-variable-prefix}card-columns: #{map.get(theme.$layouts, vertical, columns)};
+            --#{tokens.$css-variable-prefix}card-rows: #{map.get(theme.$layouts, vertical, rows)};
+            --#{tokens.$css-variable-prefix}card-areas: #{map.get(theme.$layouts, vertical, areas)};
+        }
 
-.Card--boxed.Card--horizontalReversed {
-    &:has(.CardMedia--expanded) {
-        grid-template-columns: 1fr auto;
-        grid-template-rows:
-            var(--#{tokens.$css-variable-prefix}card-padding)
-            auto
-            1fr
-            auto
-            var(--#{tokens.$css-variable-prefix}card-padding); // 2.
-
-        grid-template-areas:
-            '. media'
-            'logo media'
-            'body media'
-            'footer media'
-            '. media';
-    }
+        .Card--#{$infix}horizontal {
+            --#{tokens.$css-variable-prefix}card-columns: #{map.get(theme.$layouts, horizontal, columns)};
+            --#{tokens.$css-variable-prefix}card-rows: #{map.get(theme.$layouts, horizontal, rows)};
+            --#{tokens.$css-variable-prefix}card-areas: #{map.get(theme.$layouts, horizontal, areas)};
+        }
+
+        .Card--#{$infix}horizontalReversed {
+            --#{tokens.$css-variable-prefix}card-columns: #{map.get(theme.$layouts, horizontal-reversed, columns)};
+            --#{tokens.$css-variable-prefix}card-rows: #{map.get(theme.$layouts, horizontal-reversed, rows)};
+            --#{tokens.$css-variable-prefix}card-areas: #{map.get(theme.$layouts, horizontal-reversed, areas)};
+        }
+
+        .Card--#{$infix}vertical:has(.CardArtwork:not(:only-child), .CardMedia:not(:only-child)) {
+            --#{tokens.$css-variable-prefix}card-column-gap: 0;
+        }
+
+        :is(.Card--#{$infix}horizontal, .Card--#{$infix}horizontalReversed):has(
+                .CardArtwork:not(:only-child),
+                .CardMedia:not(:only-child)
+            ) {
+            --#{tokens.$css-variable-prefix}card-column-gap: #{theme.$gap};
+        }
+
+        // 5.c
+        .Card--boxed.Card--#{$infix}vertical {
+            &:has(.CardMedia--expanded) {
+                --#{tokens.$css-variable-prefix}card-columns: #{map.get(theme.$layouts-boxed, vertical, columns)};
+                --#{tokens.$css-variable-prefix}card-rows: #{map.get(theme.$layouts-boxed, vertical, rows)};
+                --#{tokens.$css-variable-prefix}card-areas: #{map.get(theme.$layouts-boxed, vertical, areas)};
+            }
 
-    &:has(.CardMedia:not(:only-child)) {
-        padding-left: var(--#{tokens.$css-variable-prefix}card-padding);
+            &:has(.CardMedia:not(:only-child)) {
+                --#{tokens.$css-variable-prefix}card-padding-shorthand: 0 0
```

Comment:
I wonder if this shouldn't be in theme too

Reply:
I'm not sure I understand, what do you mean?

Reply:
It uses local CSS variable `--#{tokens.$css-variable-prefix}card-padding`, I wouldn't expect that going into the theme…

Reply:
ok